src/wasi_libc.zig

```diff
@@ -45,8 +45,18 @@ pub fn emulatedLibCRFileLibName(crt_file: CRTFile) []const u8 {
     };
 }
 
-pub fn crtFileFullName(crt_file: CRTFile) []const u8 {
-    return switch (crt_file) {
+pub fn getExecModelCRTFIle(wasi_exec_model: ?std.builtin.WasiExecModel) CRTFile {
+    return if (wasi_exec_model) |model|
+        switch (model) {
+            .reactor => CRTFile.crt1_reactor_o,
+            .command => CRTFile.crt1_command_o,
+        }
+    else
+        CRTFile.crt1_o;
```

Comment:
Based on my research it looks like every WASI program is either a command or a reactor. Is it meaningful for a program to be neither? Why is crt1_o even an option?

Reply:
yeah we should remove crt1_o in favor of crt1_command_o, where the latter lets wasm-ld call ctors rather than explicitly calling in crt.c. Also I believe this reduces the complexity here about null vs default stuff.

so I removed the support for crt1_o. cc @kubkon wdyt?